python/test/unit/language/test_core.py

```diff
@@ -1654,7 +1654,7 @@ def change_value(X, BLOCK_SIZE: tl.constexpr, sem: tl.constexpr):
 
 
 @pytest.mark.interpreter
-@pytest.mark.skipif(torch.cuda.get_device_capability()[0] < 9 or is_hip(),
+@pytest.mark.skipif(not is_cuda() or torch.cuda.get_device_capability()[0] < 9 or is_hip(),
```

Comment:
```suggestion
@pytest.mark.skipif(not is_cuda() or torch.cuda.get_device_capability()[0] < 9,
```

`not is_cuda()` includes `is_hip()`

Reply:
You're right. Good catch.